src/autopas/containers/linkedCells/traversals/C01CudaTraversal.h

```diff
@@ -0,0 +1,208 @@
+/**
+ * @file C01CudaTraversal.h
+ * @author jspahl
+ * @date 11.03.2019
+ */
+
+#pragma once
+
+#include "LinkedCellTraversalInterface.h"
+#include "autopas/containers/cellPairTraversals/CellPairTraversal.h"
+#include "autopas/options/DataLayoutOption.h"
+#include "autopas/pairwiseFunctors/CellFunctor.h"
+#include "autopas/utils/CudaDeviceVector.h"
+#include "autopas/utils/ThreeDimensionalMapping.h"
+#include "autopas/utils/WrapOpenMP.h"
+#if defined(AUTOPAS_CUDA)
+#include "autopas/pairwiseFunctors/LJFunctor.h"
+#include "autopas/pairwiseFunctors/LJFunctorCuda.cuh"
+#include "autopas/utils/CudaExceptionHandler.h"
+#include "autopas/utils/CudaStreamHandler.h"
+#endif
+namespace autopas {
+
+/**
+ * This class provides the c01 traversal on the GPU.
+ *
+ * The traversal calculates all cells in parallel
+ *
+ * @tparam ParticleCell the type of cells
+ * @tparam PairwiseFunctor The functor that defines the interaction of two particles.
+ * @tparam DataLayout
+ * @tparam useNewton3
+ */
+template <class ParticleCell, class PairwiseFunctor, DataLayoutOption DataLayout, bool useNewton3>
+class C01CudaTraversal : public CellPairTraversal<ParticleCell>, public LinkedCellTraversalInterface<ParticleCell> {
```

Comment:
what is the reason, that this traversal is not using the C01BasedTraversal, even though you modified it a bit to fit cuda needs?